pkgs/development/interpreters/python/build-application.nix

```diff
@@ -0,0 +1,32 @@
+{stdenv}:
+
+{ package
+}:
+
+let
+  # Create Python environment with our package
+  env = package.python.withPackages ( ps: with ps; [ package ]);
```

Comment:
why the `with ps;` part? It looks like you're not actually using `ps`, unless they happen to have an attribute called `package` which is probably not what you want to do. Did you maybe mean `withPackages (ps: [package] ++ ps)`?